crates/indexer/src/models/token_models/v2_token_activities.rs

```diff
@@ -0,0 +1,226 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use super::{
+    token_utils::{TokenDataIdType, TokenEvent},
+    v2_token_utils::{TokenStandard, TokenV2AggregatedDataMapping, V2TokenEvent},
+};
+use crate::{schema::token_activities_v2, util::standardize_address};
+use aptos_api_types::Event as APIEvent;
+use bigdecimal::{BigDecimal, One, Zero};
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, FieldCount, Identifiable, Insertable, Serialize)]
+#[diesel(primary_key(transaction_version, event_index))]
+#[diesel(table_name = token_activities_v2)]
+pub struct TokenActivityV2 {
+    pub transaction_version: i64,
+    pub event_index: i64,
+    pub event_account_address: String,
+    pub token_data_id: String,
+    pub property_version_v1: BigDecimal,
+    pub type_: String,
+    pub from_address: Option<String>,
+    pub to_address: Option<String>,
+    pub token_amount: BigDecimal,
+    pub before_value: Option<String>,
+    pub after_value: Option<String>,
+    pub entry_function_id_str: Option<String>,
+    pub token_standard: String,
+    pub is_fungible_v2: Option<bool>,
```

Comment:
we don't keep `is_soulbound_v2`?

Reply:
yea exactly. you can think of soulbound as metadata. fungible...i can kinda track it but it really doesn't have to be here either.